streaming/base/storage/upload.py

```diff
@@ -85,8 +96,9 @@ def _validate(self, out: Union[str, Tuple[str, str]]) -> None:
         else:
             if len(out) != 2:
                 raise ValueError(''.join([
-                    f'Invalid `out` argument. It is either a string of local/remote directory ',
-                    'or a list of two strings with [local, remote].'
+                    (f'Invalid `out` argument. It is either a string'
```

Comment:
Any reason for this change ?

Reply:
Changed it again. Not sure why we needed f-string or concatenation using ''.join().